src/decor/java/com/enderio/decoration/common/util/PaintUtils.java

```diff
@@ -25,6 +21,8 @@
 import java.util.Optional;
 
 public class PaintUtils {
+    /** @deprecated This is kinda dirty, lets just do this properly */
+    @Deprecated(forRemoval = true)
```

Comment:
idk, i think this is properly as this is used in 6 classes and I don't want that piece of code everywhere

Reply:
if we are to keep something like this, it would be better in a generic core class, but I really don't think it's good practice imo